docs/docs/conditional-rendering.md

```diff
@@ -2,8 +2,6 @@
 id: conditional-rendering
 title: Conditional Rendering
 permalink: docs/conditional-rendering.html
-prev: handling-events.html
```

Comment:
You're right - I was just going through the list of add-ons docs and accidentally updated some others too. Thanks! Fixing in that follow-up PR.